### ISSUE TITLE
[Bug]: Install script on Amazon Linux 2023 fails to install native

### ISSUE THREAD

**Issue by Unaidedsteak** [netdata/netdata#17324]
### Bug description

When using the kickstart script to install the agent on an Amazon Linux 2023 machine, the native install installs the repo, but then fails to install the agent with the following:

```
Netdata                                                                                                                                                                             181  B/s | 153  B     00:00     │
Errors during downloading metadata for repository 'netdata':                                                                                                                                                        │
  - Status code: 404 for https://repo.netdata.cloud/repos/stable/amazonlinux/2023.4.20240401/x86_64/repodata/repomd.xml (IP: 172.67.72.170)                                                                         │
Error: Failed to download metadata for repo 'netdata': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried                                                                      │
Netdata Repository Config                                                                                                                                                           308  B/s | 153  B     00:00     │
Errors during downloading metadata for repository 'netdata-repoconfig':                                                                                                                                             │
  - Status code: 404 for https://repo.netdata.cloud/repos/repoconfig/amazonlinux/2023.4.20240401/x86_64/repodata/repomd.xml (IP: 104.26.9.141)                                                                      │
Error: Failed to download metadata for repo 'netdata-repoconfig': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried                                                           │
Ignoring repositories: netdata, netdata-repoconfig                                                                                                                                                                  │
Last metadata expiration check: 0:05:25 ago on Fri Apr  5 11:34:20 2024.                                                                                                                                            │
No match for argument: netdata                                                                                                                                                                                      │
Error: Unable to find a match: netdata          `

```

### Expected behavior

Netdata is successfully installed

### Steps to reproduce

1. Create Amazon Linux 2023  x86 VM
2. Run kickstart script (optionally specifying --native-only to highlight the issue)
3. See the install fails
...

### Installation method

kickstart.sh

### System info

```shell
/etc/amazon-linux-release:Amazon Linux release 2023.4.20240401 (Amazon Linux)                                                                                                                                       │
/etc/os-release:NAME="Amazon Linux"                                                                                                                                                                                 │
/etc/os-release:VERSION="2023"                                                                                                                                                                                      │
/etc/os-release:ID="amzn"                                                                                                                                                                                           │
/etc/os-release:ID_LIKE="fedora"                                                                                                                                                                                    │
/etc/os-release:VERSION_ID="2023"                                                                                                                                                                                   │
/etc/os-release:PLATFORM_ID="platform:al2023"                                                                                                                                                                       │
/etc/os-release:PRETTY_NAME="Amazon Linux 2023.4.20240401"                                                                                                                                                          │
/etc/os-release:ANSI_COLOR="0;33"                                                                                                                                                                                   │
/etc/os-release:CPE_NAME="cpe:2.3:o:amazon:amazon_linux:2023"                                                                                                                                                       │
/etc/os-release:VENDOR_NAME="AWS"                                                                                                                                                                                   │
/etc/os-release:SUPPORT_END="2028-03-15"                                                                                                                                                                            │
/etc/system-release:Amazon Linux release 2023.4.20240401 (Amazon Linux)
```

### Netdata build info

```shell
N/A
```

**Comment by Unaidedsteak** [comment]
To me it looks like your repo structure doesn't match what the script populates based on /etc/os-release.

`https://repo.netdata.cloud/repos/repoconfig/amazonlinux/2023.4.20240401/x86_64/`
should just be
`https://repo.netdata.cloud/repos/repoconfig/amazonlinux/2023/x86_64/`

**Comment by PaulSzymanski** [comment]
I have the same issue. I noticed that dnf/yum resolves `$releasever` to `2023.4.20240401`.

```bash
$ dnf config-manager --dump-variables
arch = aarch64
basearch = aarch64
awsdomain = amazonaws.com
awsregion = eu-central-1
repoguid = ...
releasever = 2023.4.20240401
```

Now, we might just be able to override the variable like so: `echo 2023 > /etc/dnf/vars/releasever`. However, this is not a practical solution, since Amazon is using the more verbose release versioning for their repos.
